source/daplink/interface/target_flash.c

```diff
@@ -39,7 +49,14 @@ typedef enum {
     STATE_ERROR
 } state_t;
 
+#ifdef TARGET_MCU_PSOC6
```

Comment:
All the target-specific logic should be removed from this file once you rebase onto #612. If there is a further need for target-specific logic in common files, we should discuss how to avoid that.

Reply:
All the target-specific logic has been removed from this file.